Add borders for floating windows in Neovim (#3603)
dense-analysis/ale: README.md

correct commands and map filename paths between different file systems. See
`:help ale-lint-other-machines` for the full documentation on how to configure
ALE to support this.

> ### 5.xxi. How can I change the borders for floating preview windows?

Reviewer: The table of contents needs updating too, but I can fix that.

> The borders for floating preview is enabled by default. You could use the
> `g:ale_floating_window_border` to control.

Reviewer: ```suggestion
Borders for floating preview windows are enabled by default. You can use the
`g:ale_floating_window_border` setting to configure them.
```

You could disable the border with an empty list.

```vim
let g:ale_floating_window_border = []
```

If the terminal supports unicode, you may use the setting to beauty the

> floating window.

Reviewer: If the terminal supports Unicode, you might try setting the value like below, to make it look nicer.

> If the terminal supports unicode, you may use the setting to beauty the
> floating window.

Reviewer: ```suggestion
  If the terminal supports Unicode, you might try setting the value like below, to make it look nicer.
```
